### DISCUSSION TITLE
displaying image in a graphics_card

### DISCUSSION THREAD

**Discussion by DeeJaayArgh** [h2oai/wave#1816] [Q&A] [answered]
I was looking in the docs about displaying an image in a ui.graphics_card and didn't find a good example.  I've been able to display a png file in an image_card but I'd like to get it in a ui.graphics_card to use as part of a scene.  Is this possible?

**Comment by mturoci** [comment, marked as answer]
Hi @DeeJaayArgh. This is currently not possible, but should be pretty simple to add. Tracked in https://github.com/h2oai/wave/issues/1817.

Just curious, do you need this for some kind of image annotation or something else?

**Comment by DeeJaayArgh** [comment]
Thanks, I was attempting to Display some text with a png icon next to it.  This can be accomplished using an image component in a ui.form_card but I was hoping to have the layout be horizontal (image to left of text) instead of vertical (text above image).  I see that there are some experimental cards, like grid_card or flex_card that could help, but this will eventually be used in production.

I'll keep my eye on #1817, the current support of SVG in h2owave is really useful.  The clock demo was helpful in getting myself familiar the concepts of stage and scene.

**Reply by mturoci** [reply]
> Display some text with a png icon next to it

I see. There are a couple of ways to do this:

* Wave has `ui.inline` to align components horizontally instead of vertically. https://wave.h2o.ai/docs/widgets/form/inline/
* `ui.text` is able to interpret HTML so `ui.text('<span>your text</span><img/>')` shall be fine as well.

> The clock demo was helpful in getting myself familiar the concepts of stage and scene.

That's good to know!

**Reply by DeeJaayArgh** [reply]
great! ui.inline is just what I needed.  I forgot that forms could display items inline too.
